source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -256,22 +269,22 @@ Now you have all the required kubeflow packages, and you can deploy the primary
    ks generate tf-job-operator tf-job-operator
    ks apply default -c tf-job-operator
 
-This creates the CustomResourceDefinition(CRD) endpoint to launch a TFJob.
+This creates the CustomResourceDefinition (CRD) endpoint to launch a TFJob.
 
 Run a TFJob
```

Comment:
@bd-dean  Is this section "Running a TFJob" specific to the Kubeflow process only? If so, then I will down-level this section heading.

Reply:
You --can-- run the TFJob in single node, but is more likely used with Kubeflow. Down-leveling is appropriate.